docs/index.md

```diff
@@ -357,7 +357,7 @@ the MkDocs IRC channel `#mkdocs` on freenode.
 [nav]: user-guide/configuration/#nav
 [discussion group]: https://groups.google.com/forum/#!forum/mkdocs
 [GitHub issues]: https://github.com/mkdocs/mkdocs/issues
-[GitHub project pages]: https://help.github.com/articles/creating-project-pages-manually/
+[GitHub project pages]: https://docs.github.com/en/enterprise-server@2.22/github/working-with-github-pages/creating-a-github-pages-site
```

Comment:
Shouldn't this be changed to `https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/creating-a-github-pages-site`?

Reply:
From the conversation and the change-set upstream, my understanding is that the URI should be changed to:

* `https://docs.github.com/github/working-with-github-pages/creating-a-github-pages-site`